custom_components/audiconnect/sensor.py

```diff
@@ -50,3 +50,8 @@ def device_class(self):
         ):
             return self._instrument.device_class
         return None
+
+    @property
+    def state_class(self):
+        """Return the state_class"""
+        return self._instrument.state_class
```

Comment:
Should we validate the state_class similar to how `device_class` is done just above this by importing STATE_CLASSES from the same?
https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/const.py

of note: looks like DEVICE_CLASSES is deprecated. I will put together a PR to fix that.

Reply:
Good hint yes we should do it that way.